src/pivot-table/components/cells/utils/get-adjuster-color.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-unsafe-enum-comparison */
+import { COLORING } from "@qlik/nebula-table-utils/lib/utils";
+import { color } from "d3-color";
+import type { StyleService } from "../../../../types/types";
+import { Colors } from "../../shared-styles";
+
+/**
+ * Converts border color to opaque color,
+ * by alpha blending the border color with the background the header or top grid background.
+ * The background is assumed to be opaque.
+ * If the background is transparent (which is the default color) it is assumed to be white
+ */
+export default function getAdjusterColor(styleService: StyleService, isHeader: boolean) {
+  const background = isHeader ? styleService.header.background : styleService.dimensionValues.background;
+  const opaqueBackground = background === Colors.Transparent ? COLORING.WHITE : background;
+  const borderColor = color(styleService.grid.border)?.rgb();
+  const backgroundColor = color(opaqueBackground)?.rgb();
+
+  if (backgroundColor && borderColor && borderColor.opacity < 1) {
+    const r = borderColor.opacity * borderColor.r + backgroundColor.r * (1 - borderColor.opacity);
```

Comment:
You could store `(1 - borderColor.opacity)` in a variable. As it's used in 3 places

Reply:
fixed